Meta/gn/build/embed_as_string.gni

```diff
@@ -1,6 +1,6 @@
-# This file introduces a template for calling embed_as_string_view.py.
+# This file introduces a template for calling embed_as_string.py.
 #
-# embed_as_string_view behaves like C++23 #embed, converting an input file into
+# embed_as_string behaves like C++23 #embed, converting an input file into
 # an AK::StringView literal rather than a C-string literal. The literal will
```

Comment:
Forgot to update this comment.

Reply:
Yeah. I think that was CLion's automated rename picking that up, I didn't pay enough attention to the surrounding text.